crates/solidity/testing/snapshots/cst_output/Expression/function_call_options_split/generated/0.8.0.yml

```diff
@@ -5,12 +5,20 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected end of input.
+    Error: Expected .
        ╭─[crates/solidity/testing/snapshots/cst_output/Expression/function_call_options_split/input.sol:1:4]
        │
      1 │ a.b{value: 0}{gas: 1}("")
-       │    │ 
-       │    ╰─ Error occurred here.
+       │    ───────────┬──────────  
+       │               ╰──────────── Error occurred here.
     ───╯
 
-Tree: null
+Tree:
+  - Expression (Rule): # 0..25 'a.b{value: 0}{gas: 1}("")'
+      - MemberAccessExpression (Rule): # 0..3 "a.b"
```

Comment:
Currently, operators don't wrap their operands in an `Expression` node.
For inlined nodes, this should be correct, but for non-inlined ones (like `Expression` here), we should be wrapping them. Otherwise, visitors won't be able to iterate on all expressions recursively.

Since this affects CST stabilization, I wonder if it is a quick/easy fix to do in this PR, or would we have to postpone for afterwards?

Reply:
I forgot we discussed this. I would rather fix this in a small PR immediately after this one.

Reply:
#508